src/core/client/admin/test/fixtures.ts

```diff
@@ -214,6 +214,11 @@ export const settings = createFixture<GQLSettings>({
   amp: false,
 });
 
+export const settingsWithMultisite = createFixture<GQLSettings>({
+  ...settings,
+  multisite: true,
+});
```

Comment:
`createFixture` accepts the base fixture and does recursive merging for you :-)

```suggestion
export const settingsWithMultisite = createFixture<GQLSettings>({
  multisite: true,
}, settings);
```